besu/src/test/java/org/hyperledger/besu/cli/ConfigurationOverviewBuilderTest.java

```diff
@@ -209,4 +210,11 @@ void setWorldStateUpdateModeJournaled() {
     final String layeredTxPoolSelected = builder.build();
     assertThat(layeredTxPoolSelected).contains("Using JOURNALED worldstate update mode");
   }
+
+  @Test
+  void setProfile() {
+    builder.setProfile(ProfileName.DEV.name());
+    final String layeredTxPoolSelected = builder.build();
```

Comment:
layeredTxPoolSelected seems an unrelated name for this variable

Reply:
Fixed